app/providers/notify_provider.rb

```diff
@@ -23,7 +23,9 @@ def call(address:, subject:, body:, reference:)
     :sending
   rescue Notifications::Client::RequestError => e
     MetricsService.failed_to_send_to_notify
-    GovukError.notify(e, tags: { provider: "notify" })
+    unless e.message.end_with?("Not a valid email address")
```

Comment:
```suggestion
    unless e.message.include?("valid email address")
```

I wonder if this allows us to be more open to the format of this message changing, but ultimately it's unfortunate that we have to rely on the human-readable string for this check, so we're already at risk of that.